resources/assets/coffee/react/beatmapset-page/scoreboard-table-row.coffee

```diff
@@ -23,26 +24,28 @@ export class ScoreboardTableRow extends React.PureComponent
     cell = "#{bn}__cell"
 
     tr
-      className: "clickable-row #{osu.classWithModifiers("#{bn}__body-row", classMods)}",
+      className: "#{osu.classWithModifiers("#{bn}__body-row", classMods)}",
 
-      td className: osu.classWithModifiers(cell, ['rank']),
-        a
-          className: "clickable-row-link #{bn}__rank"
-          href: route('scores.show', mode: score.mode, score: score.best_id)
-          "##{index+1}"
+      el @tdLink,
+        modifiers: ['rank']
+        "##{index+1}"
 
-      td className: osu.classWithModifiers(cell, ["grade"]),
+      el @tdLink,
+        modifiers: ['grade']
         div className: "score-rank score-rank--tiny score-rank--#{score.rank}"
 
-      td className: osu.classWithModifiers(cell, ["score"]),
+      el @tdLink,
+        modifiers: ['score']
         osu.formatNumber(score.score)
 
-      td className: osu.classWithModifiers(cell, ['perfect'] if score.accuracy == 1),
+      el @tdLink,
+        modifiers: ['perfect'] if score.accuracy == 1
         "#{osu.formatNumber(score.accuracy * 100, 2)}%"
 
       td className: cell,
         if score.user.country_code
           a
+            className: "#{bn}__cell-content"
```

Comment:
there's a blank space still linking to country performance page

![](https://s.myconan.net/2020-11/2020-11-20_16-48-43.png)

Reply:
Not sure if removing it or making it dead space is an improvement

Reply:
there's hover text on the flag itself so outside of it it shouldn't link to the country perf page anymore